datafusion/optimizer/src/unwrap_cast_in_comparison.rs

```diff
@@ -140,9 +138,9 @@ impl ExprRewriter for UnwrapCastExprRewriter {
                                 try_cast_literal_to_type(left_lit_value, &expr_type)?;
                             if let Some(value) = casted_scalar_value {
                                 // unwrap the cast/try_cast for the right expr
-                                return Ok(binary_expr(
+                                return Ok(datafusion_expr::binary_expr(
```

Comment:
Why is the crate name required here now?

Reply:
Because we already have a local variable `binary_expr` in line 119. I can remove this crate name here and use `expr::BinaryExpr::new()`.